abb_irb2400_support/urdf/irb2400.urdf

```diff
@@ -4,15 +4,13 @@
 <!-- |    EDITING THIS FILE BY HAND IS NOT RECOMMENDED                                 | -->
 <!-- =================================================================================== -->
 <robot name="abb_irb2400" xmlns:xacro="http://ros.org/wiki/xacro">
+  <!-- link list -->
   <link name="base_link">
     <visual>
       <origin rpy="0 0 0" xyz="0 0 0"/>
       <geometry>
-        <mesh filename="package://abb_irb2400_support/meshes/irb2400/visual/base_link.stl"/>
+        <mesh filename="package://abb_irb2400_support/meshes/irb2400/visual/base_link.dae"/>
       </geometry>
-      <material name="abb_orange">
-        <color rgba="1 0.43 0 1"/>
-      </material>
```

Comment:
Don't the collision meshes get their colours from this material definition?

It's not needed for the `visual` meshes, but might be nice to keep around (might be even nicer to use the material definitions in `abb_resources`, but we'd first have to migrate the pkg to this repository from the experimental one).